content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,67 @@
 Invoice Time Spent on Tickets (Postpaid Support Services)
 =========================================================
 
-Directly pull the billable time you have tracked on your helpdesk tickets into sales orders and
-invoices through a project task. It gives you more control over what you charge your client, and it
-is more efficient.
+Connect your Helpdesk ticket to a Project task and use either one to record timesheets that can be
+later pulled from to bill your client. This integration provides more control and transparency over
+what the client is charged for.
 
 Configuration
 =============
 
-| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk team --> Edit` and enable the options
-  *Timesheet on Ticket* and *Time Reinvoicing*.
-| Under *Timesheet on Ticket*, choose the *Project* to which tickets (and timesheets) will be
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+Click **Edit** and enable the options *Timesheet on Ticket* and *Time Reinvoicing*.
 
-.. image:: media/reinvoice_time1.png
+In the *Timesheet on Ticket* drop-down option, choose the Project to which tickets (and timesheets)
```

Comment:
We _might_ need to change these from `*italics*` to `:guilabel:` since we're talking about UI options here. Can leave for now but let's bring up with Doc Review after.

Reply:
I ended up changing these to a `:guilabel:`, we'll see what Doc Review says later.